src/commands/dev/lib/builder-cache.ts

```diff
@@ -56,10 +59,22 @@ export async function prepareBuilderDir() {
 }
 
 // Is responsible for cleaning the cache
-export async function cleanCacheDir(): Promise<void> {
+export async function cleanCacheDir(output: Output): Promise<void> {
   try {
-    const deleteCache = await cacheDirPromise;
-    await remove(deleteCache);
+    const cacheDir = await cacheDirPromise;
+    output.log(chalk`{magenta Deleting} ${cacheDir}`);
```

Comment:
well thought, with that we are indicating to the user that the chache is being erased 👌